sdks/java/core/src/main/java/org/apache/beam/sdk/options/DefaultPipelineOptionsRegistrar.java

```diff
@@ -35,6 +35,7 @@
         .add(StreamingOptions.class)
         .add(ExperimentalOptions.class)
         .add(SdkHarnessOptions.class)
+        .add(PortablePipelineOptions.class)
```

Comment:
We can register options in PortableRunnerRegistrar as we do in here
https://github.com/apache/beam/blob/master/runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkRunnerRegistrar.java

Reply:
That would make sense if PortablePipelineOptions were defined in the same module as the portable runner itself. It actually lives in the same package as the core pipeline options. As such, it is already on the classpath of anybody who links against the core SDK and should be made available there. The intent behind this was to share those command line options between _any_ runners that support the portability framework, not just the canonical `PortableRunner`.

Reply:
Sounds reasonable.